libs/langchain/langchain/indexes/_sql_record_manager.py

```diff
@@ -259,6 +259,7 @@ def list_keys(
         before: Optional[float] = None,
         after: Optional[float] = None,
         group_ids: Optional[Sequence[str]] = None,
+        limit: Optional[int] = None,
```

Comment:
should there be a pagination option too? is that easy? can be for another pr, just wondering

Reply:
We don't need offset right now, so didn't want to introduce it since offset also implies that we need to sort (which we don't want)